### ISSUE TITLE
Popover causes error when navigating away from view

### ISSUE THREAD

**Issue by thejaff2** [tochoromero/aurelia-bootstrap#12]
My view:

```
<button type="button" class="btn btn-default" aubs-popover="custom-popover.bind: stationsorterPopover; position.bind: 'bottom'; trigger.bind: 'outsideClick';">                
                    Buttontext
                </button>

                <div ref="stationsorterPopover">
                    <div class="popover-content">
                        PopoverContent (dynamic)
                    </div>
                </div>
```

When navigating away (using Aurelia router) I get message:

```
Unhandled rejection Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at AubsPopoverCustomAttribute.detached (http://localhost:9000/scripts/vendor-bundle.js:76234:31)
    at Controller.detached (http://localhost:9000/scripts/vendor-bundle.js:22370:26)
    at View.detached (http://localhost:9000/scripts/vendor-bundle.js:20452:26)
    at removeAction (http://localhost:9000/scripts/vendor-bundle.js:20745:25)
    at ViewSlot.removeAll (http://localhost:9000/scripts/vendor-bundle.js:20764:14)
    at after (http://localhost:9000/scripts/vendor-bundle.js:27598:39)
    at work (http://localhost:9000/scripts/vendor-bundle.js:27698:9)
    at http://localhost:9000/scripts/vendor-bundle.js:27712:20
    at tryCatcher (http://localhost:9000/scripts/vendor-bundle.js:3373:23)
    at Promise._settlePromiseFromHandler (http://localhost:9000/scripts/vendor-bundle.js:2590:31)
    at Promise._settlePromise (http://localhost:9000/scripts/vendor-bundle.js:2647:18)
    at Promise._settlePromiseCtx (http://localhost:9000/scripts/vendor-bundle.js:2684:10)
    at Async._drainQueue (http://localhost:9000/scripts/vendor-bundle.js:168:12)
    at Async._drainQueues (http://localhost:9000/scripts/vendor-bundle.js:173:10)
    at Async.drainQueues (http://localhost:9000/scripts/vendor-bundle.js:47:14)

```

**Comment by thejaff2** [comment]
Sorry, I was running an old version. Issue solved when running 0.1.16.

**Comment by tochoromero** [comment]
I'm glad that did it for you
